framework/PostProcessors/BasicStatistics.py

```diff
@@ -935,7 +935,8 @@ def startVector(metric):
         paramSamples[p,:] = input['targets'][param][:]
         pbWeightsList[p] = pbWeights['realization'] if param not in pbWeights['SampledVarsPbWeight']['SampledVarsPbWeight'].keys() else pbWeights['SampledVarsPbWeight']['SampledVarsPbWeight'][param]
       pbWeightsList.append(pbWeights['realization'])
-      if None in pbWeightsList:
+      #if None in pbWeightsList:
```

Comment:
Ok.  I think we can remove the comment; I don't think the replacement is too obscure.  Whatever you like best, though.